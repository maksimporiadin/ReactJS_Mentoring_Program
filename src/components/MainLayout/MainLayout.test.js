import 'jsdom-global/register';
import React from 'react';
import { MainLayout } from './index';
import { shallow } from 'enzyme';

describe('MainLayout', () => {
    let wrapper;

    it('is renders', () => {
        wrapper = shallow(<MainLayout  />);
        expect(wrapper).toMatchSnapshot();
    });
});