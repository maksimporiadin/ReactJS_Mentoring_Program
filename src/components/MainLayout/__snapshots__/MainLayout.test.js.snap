// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout is renders 1`] = `
<div
  className="main-layout-container"
>
  <div
    className="main-layout-content"
  />
  <Footer />
</div>
`;
